### PR TITLE
clarify the goal of cities #182

### PR DESCRIPTION
**Pull request by plamarque**
Cities is a concept of the WoM that is very succinctly described.
The name comes from the analogy with real estate in real world, but applied to the digital word that the WoM defines.

This PR must  to explain :
- cities are built progressively as deeds are minted
- cities allow communities to cooperate
- cities are self governed
- they can offer extra services to the members

### CODE REVIEW

**Comment by Humblespino** [comment]
The goal of cities is to provide sustainable and efficient environments that support the well-being, prosperity, and quality of life for their residents. This involves creating infrastructure, services, and policies that address the needs of the population, foster economic growth, promote social inclusion, and ensure environmental sustainability. Ultimately, cities aim to create livable spaces where people can thrive and contribute to a vibrant and harmonious community.

**Comment by Nsisong9** [comment]
In the context of Web3, cities aim to leverage decentralized technologies and blockchain to enhance transparency, efficiency, and inclusivity in governance, services, and interactions. The goal is to create decentralized, trustless systems that empower individuals and communities, fostering a more equitable and collaborative urban environment.

**Comment by Cyrushub200** [comment]
The primary goal of cities is to provide a structured living environment that is self-governed, offering additional services to its residents. Through cooperative efforts and progressive development guided by deeds, cities aim to foster community collaboration and enhance the overall well-being of their members.

And I believe Meeds DAO is an example of the goal of the cities

**Comment by KingMajesty10** [comment]
The aim is to create decentralised systems that foster a more collaborative environment that offers more services to it's residents. Through progressive development guided by deeds, cities aim to engage the total well being of members.

**Comment by omekadaniel001** [comment]
Cities can explore the potential of Web3 to improve existing services or introduce new ones. Examples include utilizing blockchain for digital identity management, streamlining government procurement processes through smart contracts, and leveraging dApps for citizen engagement and participation.

**Comment by Pandypassion** [comment]
Cities develop gradually through various deeds, fostering cooperation among communities, governing themselves, and providing additional services to their residents.

**Comment by Lamzy55** [comment]
The objective of cities is to give economical and effective conditions that help the prosperity, thriving, and personal satisfaction for their inhabitants. This includes making foundation, administrations, and arrangements that address the requirements of the populace, encourage monetary development, advance social consideration, and guarantee ecological manageability. At last, cities communities expect to make decent spaces where individuals can flourish and add to a lively and agreeable local area.

**Comment by ChristianaEt0k** [comment]
Self governance autonomy allows cities to make decisions and enact policies that are tailored to their specific needs, priorities, and circumstances.

**Comment by Zacharial** [comment]
Cities promote social cohesion and unity among their inhabitants, enabling them to address common challenges and pursue collective goals.

**Comment by JosephineEzekiel** [comment]
The goal of city is to enhance the quality of life for its residents by providing efficient infrastructure, sustainable development, and fostering a vibrant community.

**Comment by Kamchaline** [comment]
The goals of cities revolve around leveraging decentralized governance, fostering community engagement, promoting sustainability and resilience, driving innovation and economic development, and advancing equity and inclusivity in urban governance and development.

Cities aim to leverage Web3 technologies and platforms like Meeds DAO to establish decentralized governance structures. This involves empowering citizens to participate in decision-making processes, such as urban planning, resource allocation, and policy formulation, in a transparent and inclusive manner.


Cities strive to foster community engagement and collaboration through Web3 platforms like Meeds DAO. By providing tools for citizens to contribute ideas, propose initiatives, and vote on governance proposals, cities can strengthen civic participation and amplify community voices in shaping the development and management of urban spaces.

**Comment by kuvaMan** [comment]
In the context of Web3, clarifying the goal of cities could involve leveraging decentralized technologies and blockchain to reimagine urban planning, governance, and infrastructure. This could include initiatives like smart contracts for transparent and efficient public services, decentralized autonomous organizations (DAOs) for community decision-making, and tokenization of assets to enable fractional ownership of real estate and infrastructure. The goal would be to create more inclusive, sustainable, and resilient cities empowered by Web3 principles of decentralization, transparency, and interoperability.

**Comment by Isehunyime** [comment]
In this context the primary aim of a city is to create basic amenities for the support of a good life,  basic amenities such as hospitals, schools, electricity, good roads etc,
Interestingly, narrowing it down to web3 and the meeds ecosystem the city core responsibility is to provide; transparency and accountability, decentralized collaboration, remote collaboration and networking, and finally incentivized contribution and a good reward system.

**Comment by alhajipato** [comment]
In the World of Metaverse (WoM), cities are more than just pixels – they're dynamic hubs where communities build, cooperate, and self-govern. With each deed minted, these digital spaces grow, offering residents a platform for collaboration and additional services tailored to their needs. It's a digital frontier where collective empowerment shapes the virtual landscape.

**Review by godwinkelvin91 — COMMENTED** [review]
i corrected some errors and made some changes which suits some sentense

**Comment by Humblespino** [comment]
In the context of Web3, the goal of cities may extend to establishing decentralized digital ecosystems. This involves leveraging blockchain technology and decentralized protocols to create online communities that prioritize transparency, user control, and collaboration. The aim is to build digital spaces where individuals can interact, exchange value, and contribute to the development of a decentralized and interconnected web. Web3 cities aspire to embody principles of decentralization, trust, and inclusivity in the digital realm.

**Comment by Humblespino** [comment]
In the Web3 or decentralized platforms, the idea that "cities are built progressively as deeds are minted" suggests a concept where the creation and development of digital or decentralized cities occur through the generation or minting of deeds on a blockchain. Deeds in this context may represent digital assets or tokens that confer ownership or rights within the virtual city.
As deeds are minted, they could symbolize contributions, investments, or participations in the development of the digital city. These deeds might grant individuals certain privileges, influence, or ownership stakes within the decentralized ecosystem. The growth and expansion of the city, both in terms of infrastructure and community, are tied to the creation and accumulation of these digital deeds.
Essentially, this concept aligns with the decentralized nature of Web3, where community-driven actions, represented by minted deeds, play a pivotal role in shaping and expanding the digital landscape.

**Comment by Humblespino** [comment]
**cities allow communities to cooperate**

Cities provide a physical or virtual space where communities can cooperate by fostering social, economic, and cultural interactions. In a physical city, people share resources, ideas, and services, creating an environment where collaboration and synergy can thrive. Similarly, in the digital realm, cities, especially in the context of Web3, can serve as decentralized platforms that enable communities to cooperate through blockchain technology.

In a Web3 city, communities can collaborate by participating in decentralized applications (DApps), contributing to governance through decentralized autonomous organizations (DAOs), and engaging in peer-to-peer transactions facilitated by smart contracts. The decentralized nature of these platforms allows for increased transparency, trust, and inclusivity, empowering communities to work together towards common goals in a more equitable and cooperative manner.

**Comment by Humblespino** [comment]
**cities are self governed**

The concept of cities being self-governed implies that the community within a city has the ability to manage its own affairs, make decisions, and establish rules or protocols without relying heavily on external authorities. In a self-governed city, residents actively participate in the decision-making processes and contribute to the administration of local matters.
In the context of Web3 and decentralized technologies, self-governance is often facilitated through mechanisms like decentralized autonomous organizations (DAOs). These DAOs allow community members to collectively decide on policies, allocate resources, and govern the city's digital infrastructure. Smart contracts and blockchain technology contribute to the transparency and automation of these processes, ensuring that decisions are executed according to the predefined rules established by the community.

**Comment by obajemu4255** [comment]
The concept of "Cities" within the WoM platform seems well-defined and intriguing. It cleverly parallels real-world real estate with digital space, offering a framework where communities can thrive and collaborate. The progressive construction of cities through minted deeds suggests a dynamic and evolving ecosystem. Moreover, the emphasis on self-governance and additional services for members indicates a commitment to empowering and enriching the community experience. Overall, it's a promising concept that holds potential for fostering vibrant online communities within the WoM platform.

**Comment by obajemu4255** [comment]
The concept of "Cities" within the WoM platform seems well-defined and intriguing. It cleverly parallels real-world real estate with digital space, offering a framework where communities can thrive and collaborate. The progressive construction of cities through minted deeds suggests a dynamic and evolving ecosystem. Moreover, the emphasis on self-governance and additional services for members indicates a commitment to empowering and enriching the community experience. Overall, it's a promising concept that holds potential for fostering vibrant online communities within the WoM platform.

**Comment by Fruitycuti** [comment]
The goal of a city is to ensure services to provide for basic human need
Enhance resources for positive individual and community growth
Create more secure and transparent network, driving greater collaboration and value creation

**Comment by obajemu4255** [comment]
This concept of "Cities" within the WoM platform sounds intriguing. It seems to mirror the dynamics of real-world urban development in a digital context, fostering collaboration and self-governance among communities. The progressive building of cities through deeds being minted suggests a dynamic and evolving ecosystem, while the provision of extra services adds layers of value for members. Overall, it appears to be a thoughtful approach to creating decentralized digital spaces for the future of work.
Suggestion: Describe the additional services that cities can provide to their members, enhancing their overall experience within the platform.

File: en/2-deeds.md [hunk 1]
```diff
@@ -122,11 +122,12 @@ The "Mintium" model goes further than the freemium model, as the services are fr
 
 ## 2.7. Cities
 
-The World of Meeds is a virtual world where people can buy Deeds, similar to real estate properties in virtual cities. These Deeds are only available in limited quantities and can exist only if someone redeems them with the minting points they accumulate by staking.
+The World of Meeds is a virtual world where people can buy Deeds, similar to real estate properties in virtual Cities. The Deeds are only available in limited quantities within a City and can exist only if someone redeems them with the minting points they accumulate by staking.
```
Comment (Treasureth) [inline en/2-deeds.md:125]: The World of Meeds is a virtual world where people can buy Deeds, in proximity to real estate properties in virtual Cities. The Deeds are only available in short supplies within a City and can exist only if someone redeems them with the minting points they accumulate by staking.

Comment (Treasureth) [reply en/2-deeds.md:125]: Few changes were made such as:
Changing SIMILAR to PROXIMITY and changing LIMITED QUANTITIES to SHORT SUPPLIES

**Review by Treasureth — COMMENTED** [review]
I stand corrected

**Comment by Aidee-alt** [comment]
In the World of Meeds, there will be 7 cities. Each is built gradually as Deeds are minted. Initially, only the first city exists, with subsequent cities being built one after another. A city is considered complete once all its Deeds have been minted. 

Each city in World of Meeds will serve as a DAO under Meeds. This will allow Holders of Deeds in each city  to collaboratively participate in activities that will develop the city including electing a city council, voting on decisions, establishing operating rules. 

Since the cities are decentralized and self governed, deed holders can add new services they think would benefit the members of the city.

Decentralized governance in cities, allow room for creative collaboration and cooperation, as every deed holder gets to share ideas and collectively build the city they desire.

Considering these, one would conclude that the primary goal of cities in World of Meeds is to foster creativity, cooperation, and self-governance.

File: en/4-xeeds.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ However, communities that need it can hire third-party firms providing this serv
 
 In today's world, it is hard to prove a skill beforehand. When people advertise themselves on networks, they depend on social reputation. Typically, a manager or peer provides a recommendation. It requires goodwill from the relative but rarely constitutes definitive proof for the future employer.
 
-Users on their community Hubs may earn Badges and Credentials when participating. We will let them store them also in a wallet or download them as Verifiable Credentials [^7][^8]. Such credentials are proof of knowledge/know-how certified by the issuing organization.
+Users on their Community Hubs may earn Badges and Credentials when participating. We will let them store them also in a wallet or download them as Verifiable Credentials [^7][^8]. Such credentials are proof of knowledge/know-how certified by the issuing organization.
```
Comment (Nsidibeiberedem) [inline en/4-xeeds.md:19]:
```suggestion
Users may earn Badges and Credentials when participating within their Community Hubs . We will let them store them also in a wallet or download them as Verifiable Credentials [^7][^8]. Such credentials are proof of knowledge/know-how certified by the issuing organization.
```

Comment (Nsidibeiberedem) [reply en/4-xeeds.md:19]: I did a bit of rearrangement and added a new word

**Review by Nsidibeiberedem — CHANGES_REQUESTED** [review]

**Review by Nsidibeiberedem — CHANGES_REQUESTED** [review]
I rearranged the way the sentence was made and I also added a new word

**Comment by Khaymee** [comment]
the basic goals of WoMs as a decentralized community includes

Decentralization: WoM aims to reduce reliance on central authorities and intermediaries. In the context of cities, this might mean decentralized governance structures, allowing citizens to have a more direct influence on decision-making processes.

Data Ownership and Privacy: WoM emphasizes individual control over personal data. In the urban context, this could translate into citizens having greater control over their personal information and how it's used by smart city technologies.

Interoperability: WoM seeks to enable seamless interactions between different platforms and systems. In cities, this might involve creating interoperable systems for smart infrastructure, allowing various technologies to work together more efficiently.

Tokenization and Digital Assets: WoM often involves the use of cryptocurrencies and tokens. In the urban setting, this could lead to the development of tokenized systems for various services within the city, such as transportation or energy consumption.

Smart Contracts: These self-executing contracts with the terms of the agreement directly written into code could be applied to various urban processes, such as managing public services, automating billing systems, or ensuring transparency in government contracts.

Community-driven Development: WoM envisions active participation and collaboration within communities. In the urban context, this might involve citizens actively contributing to the development and improvement of their neighborhoods through digital platforms.

It's important to note that the goals of cities in the context of WoM can be dynamic and subject to ongoing development and discussions within the WoM community. For the most accurate and current information, it's advisable to refer to recent publications, discussions, and documentation within the WoM space.

**Comment by Emmy4b** [comment]
This statement suggests a process of urban development where new deeds or land grants are minted as the city expands. This pattern of growth is often seen in colonial or frontier cities, where the expansion of the city follows the establishment of new deeds, or land grants, for settlement.  The process often begins with the granting of land to influential people or families, and as the city grows, new grants are made, extending the city's boundaries. This is a common practice in many cities around the world, with a notable example being the expansion of the Roman Empire, where the granting of new land deeds was a key driver of urban development.  However, this approach can have several drawbacks. For instance, it can lead to the formation of enclaves, where the most powerful or wealthy people control large tracts of land, potentially leading to social inequality. It can also result in chaotic urban planning, with disconnected neighborhoods and inadequate infrastructure.  To prevent these issues, it's essential to implement comprehensive land management policies and urban planning strategies. These strategies should aim to create inclusive and sustainable communities, where all residents have access to essential services and

**Comment by Emmy4b** [comment]
That's an interesting concept! The idea of cities in the world of WoM sounds intriguing. It seems like it could be a way to create digital spaces or communities within the WoM.

**Comment by Patricksylv** [comment]
In the context of Meeds, the goal of cities  revolve around creating a collaborative and decentralized environment for users. This involve providing a space where community members interact, share resources and engage in various activities facilitated by the platform. The platform aim to foster cooperation, inclusivity, and community-driven initiatives, possibly leveraging decentralized technologies for governance and decision-making.

File: en/2-deeds.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ Hubs can integrate with various platforms like GitHub, Twitter, and Discord and
 Organizations can tailor the store with custom perks, including training courses and gym memberships, creating a self-reinforcing cycle of rewards and engagement. This internal store further incentivizes contributors and fosters a deeper connection with the organization's ecosystem.
 
 **Engagement and Rewards**
-Active participation contributes to community's engagement score. This metric reflects the effectiveness of engagement strategies, considering factors like user activity levels and the equitable distribution of work and rewards. This publicly auditable score entitles Hubs to a distribution of MEED tokens minted from the protocol, serving as a measure of the effectiveness of their engagement strategies.
+Active participation contributes to the community's engagement score. This metric reflects the effectiveness of engagement strategies, considering factors like user activity levels and the equitable distribution of work and rewards. This publicly auditable score entitles Hubs to a distribution of MEED tokens minted from the protocol, serving as a measure of the effectiveness of their engagement strategies.
```
[the remark this answers is not in the document]
Comment (plamarque) [reply en/2-deeds.md:66]: applied

File: en/2-deeds.md [hunk 4]
```diff
@@ -138,12 +139,18 @@ The WoM will have **7** successive phases of Deed minting that we call __Cities_
...
+
+**City Development**
+Cities strive to provide additional value to their residents beyond what individual communities can achieve independently. The concentration of resources, opportunities, and services in a City allows for community collaboration and cooperation, creating collective value. Such dynamics can lead to economic opportunities, cultural richness, educational and real-life services, and an overall improvement in the quality of life for residents.
+
+Cities may include physical counterparts to the Deeds in the real world, such as access to meeting rooms in co-working spaces, catering services, travel services, and more.
+
+Cities may also explore the potential of Web3 to improve existing services or introduce new ones, such as utilizing blockchain for digital identity management, streamlining government procurement processes through smart contracts, and leveraging dApps for citizen engagement and participation.
```
[the remark this answers is not in the document]
Comment (plamarque) [reply en/2-deeds.md:152]: applied

File: en/2-deeds.md [hunk 5]
```diff
@@ -138,12 +139,18 @@ The WoM will have **7** successive phases of Deed minting that we call __Cities_
 | 6: Kushor | 10 000 000 | Eshmum + 64 months | 6 000 000 | 3 000 000 | 999 998 | 2 |   
 | 7: Hammon | 100 000 000 | Kushor + 128 months | 60 000 000 | 30 000 000 | 9 999 999 | 1 |   
 
-A City is a way for the communities to unite and create a larger community. And as a consequence, acquire a specific identity associated with that community.
-It is also a way to provide additional value to its users, which would go beyond one community's capabilities or value created through community cooperation.
+Cities aim to bring together diverse communities to create a larger, integrated community. The purpose is to foster a sense of unity and shared identity that transcends individual communities and results in shared culture, traditions, and values.
```
[the remark this answers is not in the document]
Comment (plamarque) [reply en/2-deeds.md:142]: applied

File: en/2-deeds.md [hunk 6]
```diff
@@ -122,11 +122,12 @@ The "Mintium" model goes further than the freemium model, as the services are fr
 
 ## 2.7. Cities
 
-In the World of Meeds, Deeds are like real estate properties inside virtual cities where communities can settle.
+The World of Meeds is a virtual world where people can buy Deeds, similar to real estate properties in virtual Cities. The Deeds are only available in limited quantities within a City and can exist only if someone redeems them with the minting points they accumulate by staking.
 
-They come in limited quantities and can only exist when someone redeems them with the minting point they accumulate by staking. 
+**The 7 Cities of the WoM**
+As more people mint Deed NFTs, the Cities of the WoM are constructed progressively following a predetermined scheme that represents the successive phases of the evolution of the WoM.
```
[the remark this answers is not in the document]
Comment (plamarque) [reply en/2-deeds.md:128]: applied

File: en/2-deeds.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (plamarque) [reply en/2-deeds.md:66]: applied

File: en/2-deeds.md (hunk 1 above)
Comment (plamarque) [reply en/2-deeds.md:125]: disagreed. Deeds are in shot supplies only on the first cities, but the total supply is huge.

**Pull request merged**